packages/@webex/plugin-cc/src/request.ts

```diff
@@ -0,0 +1,48 @@
+// This file can include common request to the API. Request method can be updated in the future based on needs.
+
+import {HTTP_METHODS, WebexSDK} from './types';
+
+export default class Request {
```

Comment:
Name the class maybe HttpRequest or something else because the method and class name are the same. We can try avoiding that for better readability

Reply:
Code Updated!